eventvalidations.py

```diff
@@ -0,0 +1,14 @@
+from datetime import datetime
+
+DAYS = set(('Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'))
+
```

Comment:
as to help with the reducing of all the lines above, what if there was just once method called `validate_payload` .. that utilizes all these methods 🤔

Reply:
👍 fixed .. commented on the wrong line